tests/regression/test_mesh_generation.py

```diff
@@ -54,6 +54,80 @@ def test_unit_cube():
     assert abs(integrate_one(UnitCubeMesh(3, 3, 3)) - 1) < 1e-3
 
 
+@pytest.mark.parallel(nprocs=2)
+def test_one_element_advection():
```

Comment:
Can you rename this function `run_one_element_advection` (without the decorator) so that we can test it both in serial and in parallel with:
```python
def test_one_element_advection():
    run_one_element_advection()

@pytest.mark.parallel(nprocs=2)
def test_one_element_advection_parallel():
    run_one_element_advection()
```